apps/web/scripts/cleanTranslationsfuns.ts

```diff
@@ -0,0 +1,95 @@
+import fs from "fs-extra";
```

Comment:
No need to do this in a separate file, you can include this in `cleanTranslations.ts`